charts/newrelic-k8s-metrics-adapter/templates/apiservice/job-patch/job-patchAPIService.yaml

```
@@ -3,50 +3,48 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   namespace: {{ .Release.Namespace }}
-  name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice-patch
+  name: {{ template "common.naming.fullname" . }}-apiservice-patch
   annotations:
     "helm.sh/hook": post-install,post-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
   labels:
-    app: {{ template "newrelic-k8s-metrics-adapter.name" $ }}
-{{ include "newrelic-k8s-metrics-adapter.labels" $ | indent 4 }}
+    {{- include "common.labels" . | nindent 4 }}
 spec:
   template:
     metadata:
-      name: {{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice-patch
+      name: {{ template "common.naming.fullname" . }}-apiservice-patch
       labels:
-        app: {{ template "newrelic-k8s-metrics-adapter.name" $ }}
-{{ include "newrelic-k8s-metrics-adapter.labels" $ | indent 8 }}
+        {{- include "common.labels" . | nindent 8 }}
     spec:
-      {{- if .Values.apiServicePatchJob.image.pullSecrets }}
+      {{- with include "common.images.renderPullSecrets" ( dict "pullSecrets" (list .Values.image.pullSecrets) "context" .) }}
       imagePullSecrets:
-        {{- toYaml .Values.apiServicePatchJob.image.pullSecrets | nindent 8 }}
+        {{- . | nindent 8 }}
       {{- end }}
       containers:
         - name: patch
-          image: {{ .Values.apiServicePatchJob.image.repository }}:{{ .Values.apiServicePatchJob.image.tag }}
+          image: {{ include "common.images.image" ( dict "imageRoot" .Values.apiServicePatchJob.image "context" .) }}
           imagePullPolicy: {{ .Values.apiServicePatchJob.image.pullPolicy }}
           args:
             - patch
             - --namespace={{ .Release.Namespace }}
-            - --secret-name={{ template "newrelic-k8s-metrics-adapter.fullname" . }}-apiservice
+            - --secret-name={{ template "common.naming.fullname" . }}-apiservice
             - --apiservice-name=v1beta1.external.metrics.k8s.io
-          {{- if .Values.apiServicePatchJob.image.volumeMounts }}
+          {{- with .Values.apiServicePatchJob.volumeMounts }}
```

No need to fix this, but in the future template directives should have the same indentation as if they where values

```suggestion
            {{- with .Values.apiServicePatchJob.volumeMounts }}
```